webapp/src/components/custom_post_types/post_type/component.tsx

```diff
@@ -248,15 +272,42 @@ const Left = styled.div`
 const Right = styled.div`
     display: flex;
     flex-grow: 1;
+    justify-content: flex-end;
+    gap: 12px;
+
+    @container main (inline-size < 566px) {
+      width: 100%;
+      justify-content: space-between;
+    }
+
+    &:empty {
+      display: none;
+    }
+`;
+
+const RowDivider = styled.hr`
+    display: none;
+
+    @container main (inline-size < 566px) {
+        &&&& {
+          display: block;
+          width: 100%;
+          margin: 0;
+          border: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
```

Comment:
One more minor nit: this should be 
```
border-top: 1px solid rgba(var(--center-channel-color-rgb), 0.08);
```
Otherwise it ends up being a 2px border on the `hr` element (since it adds the border to all sides).